core/src/main/java/bio/terra/pearl/core/model/survey/RecurrenceType.java

```diff
@@ -0,0 +1,7 @@
+package bio.terra.pearl.core.model.survey;
+
+public enum RecurrenceType {
+    NONE,
+    LONGITUDINAL, // when it recurs, each instance is a new response
```

Comment:
After googling, I have come to understand that `longitudinal` is a correct term here, but it feels jargon-y and I was confused for minute because I wasn't familiar with the term. I might vote for `NEW_RESPONSE` and `UPDATE_RESPONSE` just to make it more clear exactly what will happen

Reply:
Good topic for demo.  'Longitudinal' is the term I most often hear our customers use for this sort of survey.